timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -91,79 +98,31 @@ def run(self):
             logger.error("No  Sigma rule given.")
             return "Unable to run, no rule given to the analyzer"
         rule_name = rule.get("title", "N/A")
-        problem_strings = []
         output_strings = []
```

Comment:
Similar to the `problem_strings` you only add one line to this list. You could therefore also just return the string directly without appending it to the list first.

Reply:
Talking about the `output_strings`. here.

Reply:
done